tests/AllTests.php

```diff
@@ -11,24 +11,12 @@
 
 use PHP_CodeSniffer\Util\Tokens;
 
-if (defined('PHP_CODESNIFFER_IN_TESTS') === false) {
-    define('PHP_CODESNIFFER_IN_TESTS', true);
-}
-
-if (defined('PHP_CODESNIFFER_CBF') === false) {
-    define('PHP_CODESNIFFER_CBF', false);
-}
-
-if (defined('PHP_CODESNIFFER_VERBOSITY') === false) {
-    define('PHP_CODESNIFFER_VERBOSITY', 0);
-}
+require_once __DIR__.'/bootstrap.php';
```

Comment:
This is for backwards compat, some users might have configured their own phpunit.xml which overrides the .dist and still need that. Also I didn't want to change the previous behavior, just add.